server/routes/users/auth.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const passport = require('passport');
+
+const router = express.Router();
+
+router.post('/login', passport.authenticate('local', {
+  successRedirect : '/profile',
+  failureRedirect : '/login'
+}), 
+(req, res) => {
+  res.json({ success : true });
+});
+
+router.post('/logout', passport.authenticate('local', {
+  successRedirect : '/login',
+  failureRedirect : '/logout'
+}),
+(req, res) => {
+  res.logout();
+  res.json({ success : true });
+});
+
+router.post('/register', (req, res) => {
+  
```

Comment:
This route is incomplete and the way that it is, it'll hang the server. At least return a `res.json` that says that the route isn't finished yet.